docs/getting_started.md

```diff
@@ -226,7 +226,12 @@ NAME                                                          STATUS     ROLES
 byoh-cluster-8siai8                                           Ready      master   5m   v1.23.5
 ```
 
+## Additional: Running host-agent as a systemd service
+You can use the script `hack/install-host-agent-service.sh` to start the agent as a systemd service that restarts the agent whenever the kubeconfig changes. This is done to mitigate the problem of terminating the process during the change in kubeconfig which leads to restarting the manager and killing the process. This script allows the host agent service to be restarted after process termination, and a watcher service observes the kubeconfig for changes. This script requires superuser privillages for its execution.
```

Comment:
I'd rephrase this to be how and why it would be useful if you start as a systemd service and not in terms of mitigating something

Reply:
Done. Please have a look.